packages/integration-karma/package.json

```diff
@@ -15,11 +15,11 @@
         "@lwc/engine": "1.7.10",
         "@lwc/rollup-plugin": "1.7.10",
         "@lwc/synthetic-shadow": "1.7.10",
-        "chokidar": "^3.4.1",
+        "chokidar": "^3.4.2",
         "istanbul-api": "^3.0.0",
-        "karma": "^5.1.0",
+        "karma": "^5.1.1",
         "karma-chrome-launcher": "^3.1.0",
-        "karma-coverage": "^2.0.2",
+        "karma-coverage": "^2.0.3",
         "karma-jasmine": "^3.3.1",
         "karma-sauce-launcher": "^4.1.4"
```

Comment:
@pmdartus had to revert karma-sauce-launcher update. There is an issue with that version and old browser versions. https://github.com/karma-runner/karma-sauce-launcher/issues/210

Reply:
Thanks @ravijayaramappa